using-haiku/figma-to-react.md

```diff
@@ -4,7 +4,7 @@ description: Turn any Figma design into an interactive React component with Haik
 
 # Turn any Figma Design Into an Interactive React Component with Haiku
 
-Haiku now allows you to easily bring your [Figma](/[http://figma.com](http://figma.com/) designs to life on the web by animating in Haiku and publishing as a clean React component. This tutorial will step you through the process.
```

Comment:
@funnypapaya Removing these `/`s at the front of external links doesn't seem to work locally.